### DISCUSSION TITLE
how to expand horizontal or group to the fixed window width?

### DISCUSSION THREAD

**Discussion by lucasjinreal** [emilk/egui#2693] [Q&A]
am using group and horizontal get this:
![image](https://user-images.githubusercontent.com/21303438/213730473-8c662576-7da6-4cf0-9610-df1668bfc2c5.png)

is there a way to make the group can expand to the window width?

**Comment by emilk** [comment]
[ui.set_width](https://docs.rs/egui/latest/egui/struct.Ui.html#method.set_width) should work

**Reply by lucasjinreal** [reply]
@emilk Hi, do u mean set_width to screen width? But how to get screen width dynamically?
